<html lang="en"><head><meta charset="UTF-8"><meta name="viewport" content="width=device-width,initial-scale=1"><title>Design | SugarGain | iGEM BITS Goa</title><script src="https://2020.igem.org/common/MathJax-2.5-latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script><link href="../css/content.css" rel="stylesheet"></head><body><nav id="main-nav"><div class="navbar bg-none nav-transparent"><div class="row" id="navbar-row"><a class="navbar-brand" href="/"><i class="navbar-logo"></i><span> Sugar<strong>Gain</strong></span></a><div class="controls h-100 d-flex align-items-center"><div class="align-items-center" id="themeSwitchWrapper"><i class="icon-light"></i><div id="themeSwitch"><label class="switch" for="themeSwitchInput"><input type="checkbox" id="themeSwitchInput"><span class="slider round"></span></label></div><i class="icon-dark"></i></div><button class="btn" id="menuSwitch"><div class="menu-icon"></div></button></div></div><div class="row w-100" id="progress-container-wrapper"><div id="progress-container"><div id="nav-progress-bar"></div></div></div></div><div id="fullscreenMenuWrapper"><div class="w-100 h-100" id="fullscreenMenu"><div class="container-fluid d-flex align-items-stretch pr-0 flex-column"><div class="row nav-header w-100"><div class="col-xl-3 col-lg-2"><div class="teamLogo"></div></div><div class="col-xl-6 col-lg-8"></div><div class="col-xl-3 col-lg-2"></div></div><div class="row flex-grow-1 w-100"><div class="col-xl-3 col-lg-2"></div><div class="col-xl-6 col-lg-8 d-flex justify-content-between flex-column" id="menuContent"><div class="row"><div class="col-md-6" id="nav-headings"><ul role="tablist"><li><a href="javascript:;" id="Project-tab">Project</a><ul class="mobile-nav-items"><li><a href="/Description">Description</a></li><li><a href="/Design">Design</a></li><li><a href="/Model">Model</a></li><li><a href="/Engineering">Engineering</a></li><li><a href="/Safety">Safety</a></li><li><a href="/Implementation">Implementation</a></li></ul></li><li><a href="javascript:;" id="Human_Practices-tab">Human Practices</a><ul class="mobile-nav-items"><li><a href="/Human_Practices">Human-Centered Design</a></li><li><a href="/Education">Science Communication</a></li><li><a href="/Accessibility">Accessibility</a></li></ul></li><li><a href="javascript:;" id="Parts-tab">Parts</a><ul class="mobile-nav-items"><li><a href="/Part_Collection">Part Collection</a></li><li><a href="/Parts">Parts</a></li></ul></li><li><a href="javascript:;" id="Journal-tab">Journal</a><ul class="mobile-nav-items"><li><a href="/Notebook">Notebook</a></li><li><a href="/Experiments">Experiments</a></li><li><a href="/Appendix">Appendix</a></li></ul></li><li><a href="javascript:;" id="Team-tab">Team</a><ul class="mobile-nav-items"><li><a href="/Team">Team</a></li><li><a href="/Attributions">Attributions</a></li><li><a href="/Collaborations">Collaborations</a></li></ul></li><li><a href="javascript:;" id="Contribution-tab">Contribution</a><ul class="mobile-nav-items"><li><a href="/Contribution">Overview</a></li><li><a href="/Software">Software</a></li></ul></li><li><a href="javascript:;" id="Judging-tab">Judging</a><ul class="mobile-nav-items"><li><a href="/Judging">Judging</a></li></ul></li><li><a href="javascript:;" id="Other_Areas-tab">Other Areas</a><ul class="mobile-nav-items"><li><a href="/Dance">Dance</a></li><li><a href="/Policy">Policy</a></li><li><a href="/Entrepreneurship">Entrepreneurship</a></li></ul></li></ul></div><div class="col-6" id="nav-items"><div class="tab-content"><div class="tab-pane" id="Project-pane"><ul><li><a href="/Description">Description</a></li><li><a href="/Design">Design</a></li><li><a href="/Model">Model</a></li><li><a href="/Engineering">Engineering</a></li><li><a href="/Safety">Safety</a></li><li><a href="/Implementation">Implementation</a></li></ul></div><div class="tab-pane" id="Human_Practices-pane"><ul><li><a href="/Human_Practices">Human-Centered Design</a></li><li><a href="/Education">Science Communication</a></li><li><a href="/Accessibility">Accessibility</a></li></ul></div><div class="tab-pane" id="Parts-pane"><ul><li><a href="/Part_Collection">Part Collection</a></li><li><a href="/Parts">Parts</a></li></ul></div><div class="tab-pane" id="Journal-pane"><ul><li><a href="/Notebook">Notebook</a></li><li><a href="/Experiments">Experiments</a></li><li><a href="/Appendix">Appendix</a></li></ul></div><div class="tab-pane" id="Team-pane"><ul><li><a href="/Team">Team</a></li><li><a href="/Attributions">Attributions</a></li><li><a href="/Collaborations">Collaborations</a></li></ul></div><div class="tab-pane" id="Contribution-pane"><ul><li><a href="/Contribution">Overview</a></li><li><a href="/Software">Software</a></li></ul></div><div class="tab-pane" id="Judging-pane"><ul><li><a href="/Judging">Judging</a></li></ul></div><div class="tab-pane" id="Other_Areas-pane"><ul><li><a href="/Dance">Dance</a></li><li><a href="/Policy">Policy</a></li><li><a href="/Entrepreneurship">Entrepreneurship</a></li></ul></div></div></div></div><div class="row"><div id="contact-links"><ul><li><a href="https://instagram.com/igem_bits" target="_blank" rel="noopener"><i class="insta"></i><span>Instagram</span></a></li><li><a href="https://twitter.com/igembitsgoa" target="_blank" rel="noopener"><i class="twitter"></i><span>Twitter</span></a></li><li><a href="mailto:igembitsgoa@gmail.com" target="_blank" rel="noopener"><i class="email"></i><span>Email</span></a></li><li><a href="/Sample"><i class="sample"></i><span>Sample</span></a></li></ul></div></div></div><div class="col-xl-3 col-lg-2"></div></div><div class="row nav-footer w-100"><div class="col-xl-3 col-lg-2"></div><div class="col-xl-6 col-lg-8"></div><div class="col-xl-3 col-lg-2"></div></div></div></div></div></nav><style>header {
    background-image: url( ../assets/img/Design--jean-wimmerlin.jpg);
}</style><header class="d-flex justify-content-center"><div class="custom-container d-flex align-items-end justify-content-between"><div id="heading"><h1>Design</h1></div><a href="https://unsplash.com/photos/tjZ-g9VnT7U" target="_blank" rel="noopener"><span id="bg-attribution" title="Photo by Jean Wimmerlin on Unsplash" data-placement="top"></span></a></div></header><main class="theme-project"><div class="custom-container"><div class="row"><div class="sidebar"><div class="nav" id="contents"><ul></ul></div></div><div class="content"><article><h1>Overview</h1><p>We aim to target invertase, the enzyme present in the sugarcane stem which is responsible for the degradation of sucrose into glucose and fructose. The project has been structured into the following major modules:</p><ul><li>A fructose-regulated anti-invertase construct</li><li>An atmospherically-regulated kill switch</li><li>A novel polymer inoculant</li></ul><p>Each aspect of the project design has been explained in the proceeding sections.</p><h1>Fructose-regulated Construct</h1><p>The primary product of the action of the invertase enzyme is fructose and glucose. Hence the concentration of fructose inside the sugarcane stem is proportional to the activity of invertase and consequently the amount of invertase present. We employed this dynamic concentration of fructose to modulate the expression of anti-invertase.</p><p>We designed a genetic construct based on a biosensor created by Paris-Saclay (<a href="http://parts.igem.org/Part:BBa_K2448032">BBa_K2448032</a>). A biosensor is a module that combines biological molecules as the recognition element with a physical transducer or repressor and outputs quantitative data corresponding to the biomolecule’s concentration <a href="#citation1">Park, M et al., 2013</a>. Generally, these specific interactions between the ligand and the transducer are used to determine secretion or production rates in the form of an electrochemical and/or optical signal. In our case, we coupled the biosensor to the expression of anti-invertase.</p><h2>pFruB - Cra (FruR) Regulation</h2><p>The enterobacterial catabolite repressor/activator (Cra) protein or the FruR protein is a pleiotropic regulator that controls expression of a large number of metabolic genes in response to the flux of glycolytic intermediates in various natural biological systems. The regulator is able to interact with specifically fructose-1-phosphate and regulate the action of pFruB promoter (which is a common promoter usually found in the fructose operon) <a href="#citation2">Pereira, L.F.M. et al., 2016</a> <a href="#citation3">Chavarría, M et al., 2014</a>.</p><h2>Mechanism</h2><p>The FruR expression is under an IPTG-induced pLac promoter <a href="http://parts.igem.org/Part:BBa_K3156000">(BBa_K3156000)</a> which is induced before packaging the bacteria in our polymer inoculant. FruR is a transcription factor with an affinity for fructose-1-phosphate which is produced from the D-fructose formed after invertase action. The FruR gene <a href="http://parts.igem.org/Part:BBa_K2448009">(BBa_K2448009)</a> encoding for the FruR protein prevents the transcription of the regulated promoters. pFruB <a href="http://parts.igem.org/Part:BBa_K2448017">(BBa_K2448017)</a> is the promoter region following FruR and is repressed by the FruR transcription factor, in the absence of D-Fructose. This prevents any further transcription, and no anti-invertase protein is produced. If D-Fructose is present in the cell, the FruR transcription factor will bind preferentially to it and thus be inactivated. This means that the repression of the related promoter pFruB will be released, enabling the transcription of the anti-invertase protein which can then act on invertase and prevent the inversion of sucrose.</p><blockquote><p>GIF</p></blockquote><h1>Kill Switch</h1><p>Advances in synthetic biology raise increasingly essential questions about biocontamination and biological containment. It becomes imperative that genetically-engineered organisms are not deleterious to the ecosystem and that scientists have adequate control over their production and multiplication. Certain safeguards are developed to ensure all possible safety — which are then introduced into the genetically modified organism — that compels the organism, in our case the bacteria, to rely on a “survival” factor to maintain viability in the population. Kill switches when introduced into the genetic material results in cellular death under certain conditions, such as the activation of a “trigger” caused by environmental changes. Thus helping us effectively control the proliferation of the genetically-engineered microorganism.</p><p>We intend to introduce our strain of bacteria into the sugarcane matrix to synthesise anti-invertase. Considering the close proximity of the system with both agricultural environments and human consumables, it is indispensable that our bacteria include a lethality mechanism. We propose to use a <strong>modified type II ccdA-ccdB toxin-antitoxin system</strong> as our kill switch that is activated upon exposure to atmospheric concentrations of oxygen. A change in the oxygen concentration in the environment results in a sharp increase in the amount of the toxin protein being synthesised which disturbs the already established equilibrium between the toxin and antitoxin, effectively killing the cell, and thus preventing any contamination by stray microbes.</p><h2>Type II Toxin-Antitoxin Systems</h2><p>Bacterial toxin-antitoxin systems were characterised in the 1980s as molecular systems encoded in plasmids that ensured the persistence of a plasmid in the host lineage during replication by making the cells “addicted” to the plasmid so that the only daughter cells that survived were the ones that contained the plasmid. This phenomenon is also known as post-segregational killing. Most prokaryotic chromosomes contain a number of toxin-antitoxin modules that consist of a pair of genes that encode for two components: a stable toxin protein and its unstable, labile cognate antitoxin. Toxins are always proteins, but the antitoxins can either be RNAs (in type I and III systems) or proteins (in type II systems) <a href="#citation4">Syed, M. &amp; Lévesque, C., 2012</a>.</p><p>Type II toxin-antitoxin systems use a protein antitoxin to deactivate the toxin protein via protein-protein interaction. All type II antitoxins are dual-function, two-domain proteins that consist of a protein-protein interaction domain and a DNA-binding domain. The antitoxin binds to the toxin and inhibits its activity, and the stable TA complex binds to the operator of the corresponding <dfn>TAS~Toxin-Antitoxin System</dfn> operon via the DNA-binding domain of the antitoxin and represses its transcription <a href="#citation5">Makarova et al., 2009</a> <a href="#citation6">Tam &amp; Kline, 1989</a>.</p><p>As our product would be introduced into a plant which is later converted into a consumable item, we decided to use the ccdA-ccdB toxin-antitoxin system as the kill switch for its relative non-toxicity towards humans and more generally, mammalian cells <a href="#citation7">Reschner et al., 2013</a>.</p><h2>ccdA-ccdB System</h2><p>In the ccdA-ccdB system, the two genes are organised in an operon in which the antitoxin ccdA gene is upstream of the toxin ccdB gene. ccdA prevents ccdB toxicity by forming a tight ccdA-ccdB complex <a href="#citation8">Afif et al., 2001</a>. ccdB inhibits DNA replication by poisoning DNA gyrase; more specifically, ccdB prevents DNA gyrase from disentangling DNA from replication resulting in double-stranded breaks in the DNA <a href="#citation9">Dao-Thi et al., 2005</a>. ccdA and ccdB are able to form complexes of various stoichiometries depending on their molar ratios.</p><p>Our kill switch is a modified version of the ccdA-ccdB TA system <a href="https://parts.igem.org/Part:BBa_P1010">(BBa_P1010)</a>, where the ccdB (toxin) CDS is placed under a T7 <a href="https://parts.igem.org/Part:BBa_I719005">(BBa_I719005)</a> constitutive promoter, and the ccdA (antitoxin) CDS is placed under an FNR promoter <a href="http://parts.igem.org/Part:BBa_K1123000">(BBa_K1123000)</a> whose activity is controlled by the concentration of oxygen present in the environment.</p><h2>FNR Promoter</h2><p>This part which we used is a standard FNR promoter. The promoter is used in <em>E. coli</em> strains to induce protein expression in anaerobic environments. It is natively used to induce metabolic processes when the <em>E. coli</em> enter anaerobic environments. The part is often used to express any protein sequence placed behind it in such environmental conditions. On entering a hypoxic environment, the formation of \( [ 4Fe-4S]^{2+} \) dimers is stimulated within the FNR-producing bacteria. These dimers are the transcription activators of the FNR promoter. They are able to bind to the promoter, thus allowing it to become active. Important to note is that the biobrick (number) is a tandem promoter meaning that two \( [ 4Fe-4S]^{2+} \) dimers are able to bind to the promoter. This tandem effect enhances the induction effect of the promoter, ultimately leading to a higher expression rate. $$ $$</p><p>As the environment inside sugarcane is hypoxic compared to the external atmosphere, we chose to place the ccdA antitoxin under the FNR promoter. The mechanism of the kill switch has been explained in the next section.</p><h2>Mechanism</h2><p>The kill switch function is best explained with the help of two different cases: the first where the bacteria is present in a sufficiently oxygenated environment, and the second where it is present in a hypoxic one.</p><h3>Hypoxic Environment</h3><blockquote><p>GIF</p></blockquote><p>The ccdB toxin is under a constitutive promoter which constantly expresses the toxin inside the cell. Due to prevalant hypoxic conditions inside the sugarcane <a href="#citation10">Bieleski, R. L., 1960</a> conditions, the FNR promoter will be active and hence the ccdA antitoxin will also be produced in the cell. Since both ccdA and ccdB are present in the cell, they form a highly stable toxin-antitoxin complex. This way the bacteria is able to survive inside the sugarcane stem.</p><h3>Aerobic Environment</h3><blockquote><p>GIF</p></blockquote><p>In this case, too, ccdB is expressed constitutively with the help of the T7 promoter; however, due to the presence of oxygen, the FNR promoter is switched off. This leads to an increase in the levels of ccdB in the cell which in turn leads to a complete halt in cell division and death. This ensures that the bacteria will not be able to survive outside the sugarcane environment.</p><h1>Polymer Inoculant</h1><p>We have devised a novel polymer inoculant to create an easy-to-use, robust and organic delivery mechanism of our bacteria. The inoculant consists of a Carboxymethylcellulose based polymer, bentonite suspending agent and sorbic acid preservative. We have also optimized the viscosity to minimize the innoculant’s toxicity, and ensured that it is close to that of water for efficient transport. This inoculant would be supplied to the farmer and can be used through an injector mechanism. In this manner the sugarcane’s inherent transport mechanism due to transpirational pull is leveraged for the uniform distribution of the bacteria through the plant.</p><p>To ensure large scale manufacturing of this inoculant we have constructed a process flow diagram to emulate an industrial setting. The assembly consist of a simple bioreactor, and a simple mixing tank where we mix in our inoculant ingredients. The bioreactor will be designed to be anaerobic so as to consider the killswitch. Even during the mixing process care needs to be taken that minimum aeration takes place. Through mass balance calculations we were able to compute the concentration of bacterial cells needed in the inoculant to ensure effective inactivity of invertase.</p></article><div class="d-flex justify-content-between my-5"><a class="button prev flex justify-content-end long" href="/Description"><span class="circle" aria-hidden><span class="icon arrow"></span></span><span class="button-text">Description</span></a><a class="button next short" href="/Model"><span class="circle" aria-hidden><span class="icon arrow"></span></span><span class="button-text">Model</span></a></div><article id="references"><h1>References</h1><ol><li id="citation1"><p class="author">Park, M., Tsai, S.-L., &amp; Chen, W. (2013).</p><cite>Microbial Biosensors: Engineered Microorganisms as the Sensing Machinery.</cite><p><span class="journalTitle">Sensors</span> <span class="journalInfo">13(5), 5777-5795.</span></p><a class="in-text" href="https://doi.org/10.3390/s130505777" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=Microbial Biosensors: Engineered Microorganisms as the Sensing Machinery." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext1">Back to text</a></li><li id="citation2"><p class="author">Pereira, L. F. M., Ferreira, V. M., Oliveira, N. G., Sarmento, P. L. V. S., Endres, L. &amp; Teodoro, I. (2017).</p><cite>Sugars levels of four sugarcane genotypes in different stem portions during the maturation phase.</cite><p><span class="journalTitle">Annals of the Brazilian Academy of Sciences.</span> <span class="journalInfo">89(2), 1231-1242.</span></p><a class="in-text" href="https://scholar.google.com/scholar?q=Sugars levels of four sugarcane genotypes in different stem portions during the maturation phase." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext2">Back to text</a></li><li id="citation3"><p class="author">Chavarr a, M., Durante-Rodr guez, G., Krell, T., Santiago, C., Brezovsky, J., Damborsky, J., &amp; de Lorenzo, V. (2014).</p><cite>Fructose 1-phosphate is the one and only physiological effector of the Cra (FruR) regulator ofPseudomonas putida.</cite><p><span class="journalTitle">FEBS Open Bio</span> <span class="journalInfo">4(1), 377-386.</span></p><a class="in-text" href="https://doi.org/10.1016/j.fob.2014.03.013" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=Fructose 1-phosphate is the one and only physiological effector of the Cra (FruR) regulator ofPseudomonas putida." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext3">Back to text</a></li><li id="citation4"><p class="author">Syed, M. A., &amp; L vesque, C. M. (2012).</p><cite>Chromosomal bacterial type II toxin-antitoxin systems.</cite><p><span class="journalTitle">Canadian Journal of Microbiology</span> <span class="journalInfo">58(5), 553-562.</span></p><a class="in-text" href="https://doi.org/10.1139/w2012-025" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=Chromosomal bacterial type II toxin-antitoxin systems." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext4">Back to text</a></li><li id="citation5"><p class="author">Makarova, K. S., Wolf, Y. I., &amp; Koonin, E. V. (2009).</p><cite>Comprehensive comparative-genomic analysis of Type 2 toxin-antitoxin systems and related mobile stress response systems in prokaryotes.</cite><p><span class="journalTitle">Biology Direct</span> <span class="journalInfo">4(1), 19.</span></p><a class="in-text" href="https://doi.org/10.1186/1745-6150-4-19" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=Comprehensive comparative-genomic analysis of Type 2 toxin-antitoxin systems and related mobile stress response systems in prokaryotes." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext5">Back to text</a></li><li id="citation6"><p class="author">Tam, J. E., &amp; Kline, B. C. (1989).</p><cite>The F plasmid ccd autorepressor is a complex of CcdA and CcdB proteins.</cite><p><span class="journalTitle">Molecular and General Genetics MGG</span> <span class="journalInfo">219(1-2), 26-32.</span></p><a class="in-text" href="https://doi.org/10.1007/BF00261153" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=The F plasmid ccd autorepressor is a complex of CcdA and CcdB proteins." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext6">Back to text</a></li><li id="citation7"><p class="author">Reschner, A., Scohy, S., Vandermeulen, G., Daukandt, M., Jacques, C., Michel, B., Nauwynck, H., Xhonneux, F., Préat, V., Vanderplasschen, A. &amp; Szpirer, C. (2013).</p><cite>Use of Staby® technology for development and production of DNA vaccines free of antibiotic resistance gene.</cite><p><span class="journalTitle">Human Vaccines &amp; Immunotherapeutics</span> <span class="journalInfo">9(10), 2203-2210.</span></p><a class="in-text" href="https://doi.org/10.4161/hv.25086" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=Use of Staby® technology for development and production of DNA vaccines free of antibiotic resistance gene." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext7">Back to text</a></li><li id="citation8"><p class="author">Afif, H., Allali, N., Couturier, M., &amp; Van Melderen, L. (2001).</p><cite>The ratio between CcdA and CcdB modulates the transcriptional repression of the ccd poison-antidote system.</cite><p><span class="journalTitle">Molecular Microbiology</span> <span class="journalInfo">41(1), 73-82.</span></p><a class="in-text" href="https://doi.org/10.1046/j.1365-2958.2001.02492.x" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=The ratio between CcdA and CcdB modulates the transcriptional repression of the ccd poison-antidote system." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext8">Back to text</a></li><li id="citation9"><p class="author">Dao-Thi, M.-H., Van Melderen, L., De Genst, E., Afif, H., Buts, L., Wyns, L., &amp; Loris, R. (2005).</p><cite>Molecular Basis of Gyrase Poisoning by the Addiction Toxin CcdB.</cite><p><span class="journalTitle">Journal of Molecular Biology</span> <span class="journalInfo">348(5), 1091-1102.</span></p><a class="in-text" href="https://doi.org/10.1016/j.jmb.2005.03.049" target="_blank" rel="noopener">CrossRef</a><a class="in-text" href="https://scholar.google.com/scholar?q=Molecular Basis of Gyrase Poisoning by the Addiction Toxin CcdB." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext9">Back to text</a></li><li id="citation10"><p class="author">Bieleski R. L. (1960).</p><cite>The physiology of sugar-cane. II.</cite><p><span class="journalTitle">Australian Journal of Biological Science.</span> <span class="journalInfo">13, 203-220.</span></p><a class="in-text" href="https://scholar.google.com/scholar?q=The physiology of sugar-cane. II." target="_blank" rel="noopener">Google Scholar</a><a class="in-text" href="#intext10">Back to text</a></li></ol></article></div></div></div></main><footer><div id="footerNav"><div class="container"><div class="row justify-content-between"><div class="col-sm-6 col-md-4 col-lg-3 py-2"><a class="footer-heading">Project</a><ul><li class="py-1"><a href="/Description">Description</a></li><li class="py-1"><a href="/Design">Design</a></li><li class="py-1"><a href="/Model">Model</a></li><li class="py-1"><a href="/Engineering">Engineering</a></li><li class="py-1"><a href="/Safety">Safety</a></li><li class="py-1"><a href="/Implementation">Implementation</a></li></ul></div><div class="col-sm-6 col-md-4 col-lg-3 py-2"><a class="footer-heading">Human Practices</a><ul><li class="py-1"><a href="/Human_Practices">Human-Centered Design</a></li><li class="py-1"><a href="/Education">Science Communication</a></li><li class="py-1"><a href="/Accessibility">Accessibility</a></li></ul></div><div class="col-sm-6 col-md-4 col-lg-3 py-2"><a class="footer-heading">Parts</a><ul><li class="py-1"><a href="/Part_Collection">Part Collection</a></li><li class="py-1"><a href="/Parts">Parts</a></li></ul></div><div class="col-sm-6 col-md-4 col-lg-3 py-2"><a class="footer-heading">Journal</a><ul><li class="py-1"><a href="/Notebook">Notebook</a></li><li class="py-1"><a href="/Experiments">Experiments</a></li><li class="py-1"><a href="/Appendix">Appendix</a></li></ul></div><div class="col-sm-6 col-md-4 col-lg-3 py-2"><a class="footer-heading">Team</a><ul><li class="py-1"><a href="/Team">Team</a></li><li class="py-1"><a href="/Attributions">Attributions</a></li><li class="py-1"><a href="/Collaborations">Collaborations</a></li></ul></div><div class="col-sm-6 col-md-4 col-lg-3 py-2"><a class="footer-heading">Contribution</a><ul><li class="py-1"><a href="/Contribution">Overview</a></li><li class="py-1"><a href="/Software">Software</a></li></ul></div><div class="col-sm-6 col-md-4 col-lg-3 py-2"><a class="footer-heading">Judging</a><ul><li class="py-1"><a href="/Judging">Judging</a></li></ul></div><div class="col-sm-6 col-md-4 col-lg-3 py-2"><a class="footer-heading">Other Areas</a><ul><li class="py-1"><a href="/Dance">Dance</a></li><li class="py-1"><a href="/Policy">Policy</a></li><li class="py-1"><a href="/Entrepreneurship">Entrepreneurship</a></li></ul></div></div></div></div><div id="footerBrand"><div class="container"><div class="row justify-content-center"><div id="footerTeamLogo"></div><div id="BITSLogo"></div></div><p>iGEM BITS Goa</p><p>Birla Institute of Technology and Science, Pilani</p><p>K K Birla Goa Campus</p><p>NH-17B, Sancoale, Goa, India</p><div class="row justify-content-center" id="footerIcons"><a class="gmail" href="mailto:igembitsgoa@gmail.com"></a><a class="insta" href="https://instagram.com/igem_bits" target="_blank" rel="noopener"></a><a class="twitter" href="https://twitter.com/igembitsgoa" target="_blank" rel="noopener"></a></div></div></div></footer><script src="../content.bundle.js"></script></body></html>